workspace/lib/jenkinhelper/jenkin.rb

```diff
@@ -0,0 +1,46 @@
+require 'uri'
+module JenkinHelper
+  module Jenkin
+    def form_encode(string)
+      URI.encode_www_form_component string.encode(Encoding::UTF_8)
+    end
+
+    # Encode a string for use in the hiearchical part of an URL
+    def path_encode(path)
+      URI.escape(path.encode(Encoding::UTF_8))
+    end
+
+    def client
+      JenkinsApi::Client.new(
+      server_ip: config.ip,
+      username: config.username,
+      password: config.password)
+    end
+
+    def get_item_in_queue(job_name)
+      queue_result = client.api_get_request("/job/#{path_encode job_name}")['inQueue']
+      if queue_result
+        return client.api_get_request("/job/#{path_encode job_name}")['nextBuildNumber']
+      else
+        return queue_result
```

Comment:
I think that we can update
      if queue_result
         queue_result = client.api_get_request("/job/#{path_encode job_name}")['nextBuildNumber']
      return queue_result